.travis.yml

```diff
@@ -22,14 +42,28 @@ before_install:
 
     # install facter to determine operating system and release version
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install caskroom/cask/brew-cask || true ; fi
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew cask install facter || true ; fi
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew cask install facter || true ; fi    
+
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker pull $CONTAINER_LINUX_DISTRO:$VER ; fi
+
+script:
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then make tarball ; fi
+
+    - if [[ "$TRAVIS_OS_NAME" == "linux" && "$CONTAINER_LINUX_DISTRO" == "ubuntu" ]]; then
+      docker run --rm -v `pwd`:`pwd` -w `pwd` $CONTAINER_LINUX_DISTRO:$VER /bin/bash -c
+      "sudo apt-get update; sudo apt-get install -y $DEPS_UBUNTU; make tarball" ; fi
```

Comment:
use `&&` instead of `;` so that we daisy chain the commands and fail on the first one that fails.